packages/chat-headless/src/slices/conversation.ts

```diff
@@ -2,7 +2,11 @@ import { createSlice, PayloadAction } from "@reduxjs/toolkit";
 import { ConversationState } from "../models/slices/conversation";
 import { Message, MessageNotes } from "@yext/chat-core";
 
-export const initialState: ConversationState = {
+export const STATE_SESSION_STORAGE_KEY : string = "conversationState";
```

Comment:
nit: maybe yext_chat_state?

Reply:
I went with yext_chat_conversation_state since it's not the full state.